homeassistant/components/honeywell/sensor.py

```diff
@@ -0,0 +1,109 @@
+"""Support for Honeywell (US) Total Connect Comfort sensors."""
+from __future__ import annotations
+
+from homeassistant.components.honeywell.const import (
+    DOMAIN,
+    SENSOR_LOCATION_INDOOR,
+    SENSOR_LOCATION_OUTDOOR,
+)
+from homeassistant.components.sensor import STATE_CLASS_MEASUREMENT, SensorEntity
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+)
+import homeassistant.helpers.device_registry as dr
+
+
+async def async_setup_entry(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Honeywell thermostat."""
+    data = hass.data[DOMAIN][config.entry_id]
+    sensors = []
+
+    if data.device.current_temperature is not None:
+        sensors.append(
+            HoneywellUSSensor(data, SENSOR_LOCATION_INDOOR, DEVICE_CLASS_TEMPERATURE)
+        )
+    if data.device.current_humidity is not None:
+        sensors.append(
+            HoneywellUSSensor(data, SENSOR_LOCATION_INDOOR, DEVICE_CLASS_HUMIDITY)
+        )
```

Comment:
These are already part of the climate entity, should these be disabled by default?

Reply:
I can remove them. Only reason I had them in there is to make it easy to add them to lovelace dashboards without using the climate card.